qiskit/circuit/library/arithmetic/piecewise_chebyshev.py

```diff
@@ -64,7 +63,8 @@ def __init__(
     ) -> None:
         r"""
         Args:
-            f_x: the function to be approximated.
+            f_x: the function to be approximated. Constant functions should be specified
```

Comment:
So for a constant function I would call
```
PiecewiseChebychev(0.5, degree=3, etc.)
```
? If yes, could you add floats in the typehint for `f_x`? 🙂

Reply:
exactly, added float to the typehint